src/js/views/panel_view.js

```diff
@@ -375,15 +372,8 @@
 
             // update src, layout etc.
             this.render_image();
-            this.render_labels();
-            this.render_scalebar();     // also calls render_layout()
-
-            // At this point, element is not ready for Raphael svg
-            // If we wait a short time, works fine
-            var self = this;
-            setTimeout(function(){
-                self.render_shapes();
-            }, 10);
+            this.render_scalebar();     // also calls render_layout() -> render_labels()
+            this.render_shapes();
```

Comment:
the conflict with https://github.com/ome/omero-figure/pull/605 is coming from here

https://github.com/ome/omero-figure/pull/605/files#diff-fd31cdc90e2bd6fb92184b2b54474aa097d94dcbcc6ebf591b99715c6c0b0744R449-R452